pkg/blockstorage/awsebs/awsebs.go

```diff
@@ -407,13 +408,13 @@ func (s *ebsStorage) VolumeCreateFromSnapshot(ctx context.Context, snapshot bloc
 	if snapshot.Volume.VolumeType == "" || snapshot.Volume.Az == "" || snapshot.Volume.Tags == nil {
 		return nil, errors.Errorf("Required volume fields not available, volumeType: %s, Az: %s, VolumeTags: %v", snapshot.Volume.VolumeType, snapshot.Volume.Az, snapshot.Volume.Tags)
 	}
-
-	zones, err := zone.FromSourceRegionZone(ctx, s, snapshot.Region, snapshot.Volume.Az)
+	cli, _ := kube.NewClient()
```

Comment:
Is the `_` needed? If there's error, we shouldn't ignore it

Reply:
The error is not needed. If the CLI is nil we use the provider specified zones directly.

Reply:
Do we always expect to be able to get the CLI? I would hoist this one level higher when we construct ebsStorage and log an error if it's not used. 

This would even let us mock the cli in unit tests by setting the the field in the struct.